lm75bd/lm75bd.c

```diff
@@ -27,7 +27,35 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+
+  error_code_t errCode;
+
+  uint8_t pointerRegToTempRegBuf[1] = {0};
+  RETURN_IF_ERROR_CODE(i2cSendTo(devAddr, pointerRegToTempRegBuf, 1));
+
+  uint8_t tempData[2] = {0};
```

Comment:
Define a macro for the temperature register (`LM75BD_REG_TEMP`) at the top of this file below `LM75BD_REG_CONF`.